dags/probe_scraper.py

```diff
@@ -265,17 +260,40 @@
                 "AWS_ACCESS_KEY_ID": aws_access_key,
                 "AWS_SECRET_ACCESS_KEY": aws_secret_key
             },
-            # wait for upstream, ignore upstream failures, and don't run if day_check
-            # status is "skipped", i.e. still only run on specific days
-            trigger_rule="none_skipped",
             dag=dag,
             **airflow_gke_prod_kwargs,
         )
-        day_check >> check_task
-        probe_scraper_glean_repositories >> check_task
-        for glean_task in probe_scraper_glean:
-            glean_task >> check_task
-        probe_scraper_checks.append(check_task)
+        for check_name in ("check-expiry", "check-fog-expiry")
+    ]
+    dummy_branch = DummyOperator(
+        task_id=f"dummy_branch",
+        dag=dag,
+    )
+
+
+    class CheckBranchOperator(BaseBranchOperator):
+        def choose_branch(self, context):
```

Comment:
Some docs here would be helpful, e.g.
```suggestion
    class CheckBranchOperator(BaseBranchOperator):
        """
        The `choose_branch` method returns an array of task_ids
        to be executed. These tasks must be downstream of the
        branch task.
        """
        def choose_branch(self, context):
```

Reply:
yes, but I think the docs should go directly on the `choose_branch` method